src/Components/Footer/Footer.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react'
+import React from "react";
+import "./Footer.scss";
+import logo from "../../asset/copyright-logo.png";
 
 const Footer = () => {
   return (
-    <div>Footer</div>
-  )
-}
+    <div className="footer">
```

Comment:
nice work but you need to use the Footer tag for the footer instead of the div tag

Reply:
thanks bernard! just committed the updates